go/storage/mkvs/syncer/proof.go

```diff
@@ -33,21 +44,37 @@ type proofNode struct {
 
 // ProofBuilder is a Merkle proof builder.
 type ProofBuilder struct {
-	root     hash.Hash
-	subtree  hash.Hash
-	included map[hash.Hash]*proofNode
-	size     uint64
+	proofVersion uint16
```

Comment:
```suggestion
	proofVersion uint8
```

Reply:
Using `uint16` so that it is consistent with the version in the `Proof`s, which embeds our common `cbor.Versioned` helper struct that uses `uint16` for versions.